.github/workflows/199-release-make-ci-upload-release-artifacts.yml

```diff
@@ -45,6 +45,7 @@ jobs:
           credentials:
             username:  ${{ github.actor }}
             password:  ${{ secrets.GITHUB_TOKEN }}
+          options: --security-opt seccomp=unconfined
```

Comment:
from the discord: maybe "--privileged" is the right flag as it grants CAP_SYS_ADMIN: https://docs.docker.com/engine/reference/commandline/run/#:~:text=The%20%2D%2Dprivileged%20flag%20gives,like%20running%20Docker%20within%20Docker.

Reply:
Neither one worked. It may have to do with the how the `skip_if_seccomp` macro is detecting Seccomp mode. Going to close the PR.